README.md

```diff
@@ -23,3 +23,18 @@
 ## Code Quality 🌟
 
 - This project uses prettier and ESlint to increase code readability and consistency.
+
```

Comment:
You can remove this from the readme, you can add it as a description to your story though as it is good to have. We will try and keep to adding features to the readme and have components described in stories.

Reply:
sorted